packages/newspringchurchapp/src/content-single/Features/TextFeature.js

```diff
@@ -2,23 +2,47 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { TouchableOpacity } from 'react-native';
 
-import { ActionCard, BodyText } from '@apollosproject/ui-kit';
+import {
+  ActionCard,
+  BodyText,
+  TextInput,
+  styled,
+} from '@apollosproject/ui-kit';
 import ShareButton from 'newspringchurchapp/src/ui/ShareButton';
 
+const StyledTextInput = styled(({ theme }) => ({
+  height: theme.sizing.baseUnit * 6,
+  borderWidth: 0.5,
+  borderColor: theme.colors.shadows.default,
+  paddingHorizontal: theme.sizing.baseUnit * 0.5,
+  textAlignVertical: 'top',
+}))(TextInput);
+
 const TextFeature = ({ body, contentId }) => {
   const [isPressed, press] = useState(false);
+  const [notesText, setNotesText] = useState('');
   const bodyWithBlank = body.replace(/__(.*)__/gm, (match, p1) =>
     '_'.repeat(p1.length)
   );
   const bodyWithWord = body.replace(/__(.*)__/gm, (match, p1) => p1);
+  console.log(notesText);
 
   return (
     <TouchableOpacity onPress={() => press(true)}>
       <ActionCard
         icon={'play'}
-        action={<ShareButton message={body} itemId={contentId} />}
+        action={
+          <ShareButton
+            title={`${bodyWithWord} || My Notes: ${notesText.text}`}
```

Comment:
why did you replace `message` with `title` here?

Reply:
also shouldn't include that in the message if the custom notes are blank

Reply:
I changed it to title because the message wasn't actually in the shared body. I'm asking core about it.